bokeh/document.py

```diff
@@ -431,3 +469,30 @@ def validate(self):
             refs = r.references()
             root_sets.append(refs)
             check_integrity(refs)
+
+    @property
+    def session_callbacks(self):
+        return self._session_callbacks.values()
```

Comment:
if the field is `session_callbacks` I'd expect the event to be `SessionCallbackAdded` - there's more than one way to fix that, could change either the field name or the event name. I think changing the event name would reduce the total amount of code because if we have separate `periodic_callbacks` and `timeout_callbacks` fields and `PeriodicCallbackAdded`, `TimeoutCallbackAdded`, then we need twice as much code over in ClientSession and ServerSession to watch both kinds of event and load both fields.